code/modules/research/nanites/nanite_programs/healing.dm

```diff
@@ -70,9 +70,14 @@
 	rogue_types = list(/datum/nanite_program/brain_decay)
 
 /datum/nanite_program/brain_heal/check_conditions()
-	if(!host_mob.getBrainLoss())
-		return FALSE
-	return ..()
+	var/problems = FALSE
```

Comment:
Eugh. What happened to TG's coding standards?
You can axe this var declaration then replace the `problems = TRUE` lines with `return ..()`, and replace the final line in the proc with a simple `return FALSE`
There's no such thing as enough micro-optimizations, especially for things like nanites that get their procs called repeatedly en-masse. Var declarations within procs in byond have a notable performance impact, so you typically want to keep var declarations to a minimum whenever possible inside procs that get called frequently.

Reply:
I guess oranges truly ruined em good.

Reply:
I wish i knew how to these optimizations myself honestly